install-wings.sh

```diff
@@ -240,157 +249,79 @@ function check_os_comp {
-    print_error "OS not supported."
-    exit 1
-  fi
-
-  # If user has nginx
-  systemctl stop nginx || true
-
-  # Obtain certificate
-  certbot certonly --no-eff-email --email "$EMAIL" --standalone -d "$FQDN" || FAILED=true
-
-  systemctl start nginx || true
-
-  # Check if it succeded
-  if [ ! -d "/etc/letsencrypt/live/$FQDN/" ] || [ "$FAILED" == true ]; then
-    print_warning "The process of obtaining a Let's Encrypt certificate failed!"
-  fi
+apt_update() {
+  apt update -q -y && apt upgrade -y
 }
 
-function apt_update {
-  apt update -y
-  apt upgrade -y
+yum_update() {
+  yum -y update
 }
 
-function install_dep {
-  if [ "$OS" == "debian" ] || [ "$OS" == "ubuntu" ]; then
-    apt_update
-
-    # install dependencies
-    apt -y install curl
-  elif [ "$OS" == "centos" ]; then
-    if [ "$OS_VER_MAJOR" == "7" ]; then
-      yum -y update
-
-      # install dependencies
-      yum -y install curl
-    elif [ "$OS_VER_MAJOR" == "8" ]; then
-      dnf -y update
+dnf_update() {
+  dnf -y upgrade
+}
 
-      # install dependencies
-      dnf install -y curl
-    fi
-  else
-    print_error "Invalid OS."
-    exit 1
-  fi
+enable_docker(){
+  systemctl start docker
+  systemctl enable docker
 }
 
-function install_docker {
+install_docker() {
   echo "* Installing docker .."
-  if [ "$OS" == "debian" ]; then
-    # install dependencies for Docker
-    apt-get update
-    apt-get -y install \
-     apt-transport-https \
-     ca-certificates \
-     curl \
-     gnupg2 \
-     software-properties-common
-
-    # get their GPG key
-    curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
-
-    # show fingerprint to user
-    apt-key fingerprint 0EBFCD88
-
-    # add APT repo
-    add-apt-repository \
-      "deb [arch=amd64] https://download.docker.com/linux/debian \
-      $(lsb_release -cs) \
-      stable"
-
-    # install docker
-    apt-get update
-    apt-get -y install docker-ce docker-ce-cli containerd.io
-
-    # make sure it's enabled & running
-    systemctl start docker
-    systemctl enable docker
-
-  elif [ "$OS" == "ubuntu" ]; then
-    # install dependencies for Docker
-    apt-get update
+  if [ "$OS" == "debian" ] || [ "$OS" == "ubuntu" ]; then
+    # Install dependencies
     apt-get -y install \
       apt-transport-https \
       ca-certificates \
-      curl \
+      gnupg2 \
       software-properties-common
 
-    # get their GPG key
-    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-
-    # show fingerprint to user
-    apt-key fingerprint 0EBFCD88
+    # Add docker gpg key
+    curl -fsSL https://download.docker.com/linux/"$OS"/gpg | apt-key add -
 
-    # add APT repo
-    sudo add-apt-repository \
-     "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
-     $(lsb_release -cs) \
-     stable"
+    # Add docker repo
+    add-apt-repository \
+      "deb [arch=amd64] https://download.docker.com/linux/$OS \
+      $(lsb_release -cs) \
+      stable"
```

Comment:
```suggestion
    "deb [arch=amd64] https://download.docker.com/linux/$OS \
    $(lsb_release -cs) \
    stable"
```

Shouldn't the indenting be like this?

Reply:
![image](https://user-images.githubusercontent.com/26341983/98169392-d89db600-1ef4-11eb-857e-ad213df7c29c.png)

It doesn't show me that like that, but I will commit yours.